src/sanitiser.js

```diff
@@ -34,22 +40,29 @@ const sanitiseDefaultOptions = function(options) {
   options.headers = lowerHeaderKeys(options.headers);
   options.headers['user-agent'] =
     options.headers['user-agent'] || getDefaultUserAgent();
+  options.headers.templates =
+    options.headers.templates || getTemplatesInfo(config.templates);
 
   options.timeout = options.timeout || 5;
   return options;
 };
 
 module.exports = {
   sanitiseConfiguration: function(conf) {
+    const baseTemplates = ['oc-template-handlebars', 'oc-template-jade'];
+
     conf = conf || {};
     conf.components = conf.components || {};
     conf.cache = conf.cache || {};
+    conf.templates = conf.templates
+      ? _.uniq(conf.templates.concat(baseTemplates))
```

Comment:
Totally agree, btw I've added coverage for the uniq helper but we don't have any test at all for all the rest of the utils, to be address in an had hoc PR.